### ISSUE TITLE
[Bug] App crashes when trying to open notification from background after migrating to Android 14

### ISSUE THREAD

**Issue by makstheimba** [Pushwoosh/pushwoosh-flutter#86]
This issue is resolved in your Android SDK 6.7.11 - https://github.com/Pushwoosh/pushwoosh-android-sdk/issues/188

I wanted to create pull request but I can't create branches in this repository. Please up pushwoosh android sdk version in build gradle to 6.7.11 so we can start using this plugin again instead of our own fork.

Only 4 lines need to be changed in build.gradle:
```
    implementation 'com.pushwoosh:pushwoosh:6.7.11'
    implementation "com.pushwoosh:pushwoosh-firebase:6.7.11"
    implementation "com.pushwoosh:pushwoosh-huawei:6.7.11"
    implementation 'com.pushwoosh:pushwoosh-badge:6.7.11'
```

**Comment by Jaaco** [comment]
Opened a PR :)

@makstheimba btw in your fork on the main page, below the history, there is a button "Contribute", from which you can open a PR :)

**Comment by wfhm** [comment]
@Jaaco @makstheimba Thanks for reporting this, and apologies for the delay with the response. Our release pipelines got messed up, so we cannot roll out the fix quickly. I'll try to push a new version with the fix manually later today and let you know when it's done.

**Comment by enginseer-dev** [comment]
The issue was resolved in the [2.2.26](https://github.com/Pushwoosh/pushwoosh-flutter/releases/tag/2.2.26) plugin release.
